.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "useTabs": false,
+  "singleQuote": true,
+  "htmlWhitespaceSensitivity": "strict"
```

Comment:
This will only work with HTML and we don't use it with HTML files. Although I agree with this setting.

Reply:
why not use for html too?